{
  "uid" : "f5476102a51d99f",
  "name" : "Post login",
  "fullName" : "test_first.TestPOST#test_post_login",
  "historyId" : "1fd6fb68fa1472ef9c72aabb3bafa2fc",
  "time" : {
    "start" : 1700405167525,
    "stop" : 1700405167525,
    "duration" : 0
  },
  "status" : "skipped",
  "statusMessage" : "XFAIL\n\nFileNotFoundError: [Errno 2] No such file or directory: 'logs/log_2023-11-19_14-46-06.log'",
  "statusTrace" : "self = <test_first.TestPOST object at 0x7efccf882440>\n\n    @pytest.mark.xfail\n    @allure.sub_suite('POST')\n    @allure.title('Post login')\n    def test_post_login(self):\n        print('\\n\\nMethod POST: login')\n>       result_post = API.post_login(body)\n\ntest_first.py:74: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \nutils/request.py:91: in post_login\n    result_post = HttpMethods.post(post_url, body)\nutils/http_methods.py:24: in post\n    Logger.add_request(url, method='POST')\nutils/logger.py:28: in add_request\n    cls.write_log_to_file(data_to_add)\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\ncls = <class 'utils.logger.Logger'>\ndata = '\\n------\\nTest: test_first.py::TestPOST::test_post_login (call)\\nTime: 2023-11-19 14:46:07.525305\\nRequest method: POST\\nRequest URL: https://dbend.areso.pro/login\\n\\n'\n\n    @classmethod\n    def write_log_to_file(cls, data: str):\n>       with open(cls.file_name, 'a', encoding='utf=8') as logger_file:\nE       FileNotFoundError: [Errno 2] No such file or directory: 'logs/log_2023-11-19_14-46-06.log'\n\nutils/logger.py:13: FileNotFoundError",
  "flaky" : false,
  "newFailed" : false,
  "beforeStages" : [ ],
  "testStage" : {
    "status" : "skipped",
    "statusMessage" : "XFAIL\n\nFileNotFoundError: [Errno 2] No such file or directory: 'logs/log_2023-11-19_14-46-06.log'",
    "statusTrace" : "self = <test_first.TestPOST object at 0x7efccf882440>\n\n    @pytest.mark.xfail\n    @allure.sub_suite('POST')\n    @allure.title('Post login')\n    def test_post_login(self):\n        print('\\n\\nMethod POST: login')\n>       result_post = API.post_login(body)\n\ntest_first.py:74: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \nutils/request.py:91: in post_login\n    result_post = HttpMethods.post(post_url, body)\nutils/http_methods.py:24: in post\n    Logger.add_request(url, method='POST')\nutils/logger.py:28: in add_request\n    cls.write_log_to_file(data_to_add)\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\ncls = <class 'utils.logger.Logger'>\ndata = '\\n------\\nTest: test_first.py::TestPOST::test_post_login (call)\\nTime: 2023-11-19 14:46:07.525305\\nRequest method: POST\\nRequest URL: https://dbend.areso.pro/login\\n\\n'\n\n    @classmethod\n    def write_log_to_file(cls, data: str):\n>       with open(cls.file_name, 'a', encoding='utf=8') as logger_file:\nE       FileNotFoundError: [Errno 2] No such file or directory: 'logs/log_2023-11-19_14-46-06.log'\n\nutils/logger.py:13: FileNotFoundError",
    "steps" : [ {
      "name" : "Add request",
      "time" : {
        "start" : 1700405167525,
        "stop" : 1700405167525,
        "duration" : 0
      },
      "status" : "broken",
      "statusMessage" : "FileNotFoundError: [Errno 2] No such file or directory: 'logs/log_2023-11-19_14-46-06.log'\n",
      "statusTrace" : "  File \"/opt/hostedtoolcache/Python/3.10.13/x64/lib/python3.10/site-packages/allure_commons/_allure.py\", line 192, in impl\n    return func(*a, **kw)\n  File \"/home/runner/work/API/API/utils/logger.py\", line 28, in add_request\n    cls.write_log_to_file(data_to_add)\n  File \"/home/runner/work/API/API/utils/logger.py\", line 13, in write_log_to_file\n    with open(cls.file_name, 'a', encoding='utf=8') as logger_file:\n",
      "steps" : [ ],
      "attachments" : [ ],
      "parameters" : [ {
        "name" : "url",
        "value" : "'https://dbend.areso.pro/login'"
      }, {
        "name" : "method",
        "value" : "'POST'"
      } ],
      "attachmentsCount" : 0,
      "shouldDisplayMessage" : true,
      "stepsCount" : 0,
      "hasContent" : true
    } ],
    "attachments" : [ {
      "uid" : "5edf3e139b956fb1",
      "name" : "stdout",
      "source" : "5edf3e139b956fb1.txt",
      "type" : "text/plain",
      "size" : 51
    } ],
    "parameters" : [ ],
    "attachmentsCount" : 1,
    "shouldDisplayMessage" : true,
    "stepsCount" : 1,
    "hasContent" : true
  },
  "afterStages" : [ ],
  "labels" : [ {
    "name" : "subSuite",
    "value" : "POST"
  }, {
    "name" : "epic",
    "value" : "POST REQUESTS"
  }, {
    "name" : "suite",
    "value" : "REQUESTS POST"
  }, {
    "name" : "tag",
    "value" : "@pytest.mark.xfail"
  }, {
    "name" : "host",
    "value" : "fv-az1435-256"
  }, {
    "name" : "thread",
    "value" : "1914-MainThread"
  }, {
    "name" : "framework",
    "value" : "pytest"
  }, {
    "name" : "language",
    "value" : "cpython3"
  }, {
    "name" : "package",
    "value" : "test_first"
  }, {
    "name" : "resultFormat",
    "value" : "allure2"
  } ],
  "parameters" : [ ],
  "links" : [ ],
  "hidden" : false,
  "retry" : false,
  "extra" : {
    "severity" : "normal",
    "retries" : [ ],
    "categories" : [ ],
    "tags" : [ "@pytest.mark.xfail" ]
  },
  "source" : "f5476102a51d99f.json",
  "parameterValues" : [ ]
}